metrics/src/main/java/io/opencensus/metrics/MetricProducer.java

```diff
@@ -14,14 +14,14 @@
  * limitations under the License.
  */
 
-package io.opencensus.metrics.export;
+package io.opencensus.metrics;
```

Comment:
Why move `MetricProducer` to `metrics`, while `MetricProducerManager` is still under `metrics.export`?

Reply:
Chatted offline, this is to avoid circular dependency.